src/cephadm/cephadm

```diff
@@ -3601,13 +3601,12 @@ def prepare_dashboard(
     fw.open_ports([port])
     fw.apply_rules()
 
-    logger.info('Ceph Dashboard is now available at:\n\n'
-                '\t     URL: https://%s:%s/\n'
-                '\t    User: %s\n'
-                '\tPassword: %s\n' % (
-                    get_fqdn(), port,
-                    ctx.initial_dashboard_user,
-                    password))
+    logger.info('Ceph Dashboard is now available at:\n\n')
+    # let's print the dashboard infos to stdout and make it compatible to yaml:
```

Comment:
```suggestion
    logger.info('Ceph Dashboard is now available at:\n\n %s' % out)
    # let's print the dashboard infos to stdout and make it compatible to yaml:
```

add the context (url, user, etc) to the log msg? 

this has an unfortunate side effect where the log message to stderr (unbuffered) will appear much earlier in the console output than the below print to stdout (buffered) ...

Reply:
I'd like to avoid printing it to stderr and stdout at the same time. Without a `--yaml` flag, I opted for only printing it to stdout. 

> this has an unfortunate side effect where the log message to stderr (unbuffered) will appear much earlier in the console output than the below print to stdout (buffered) ...

well, there is `sys.stdout.flush()` fortunately!